### DISCUSSION TITLE
quale sistema di irrigazione si integra meglio in HA?

### DISCUSSION THREAD

**Discussion by reclus23** [indomus/forum#2963] [HUB: Home Assistant]
ciao, secondo voi qual è il sistema di irrigazione, tra quelli segnalati sul sito di HA, che si integra meglio?  Non ho particolari esigenze. Canale di irrigazione singolo, quindi partenza ad un certo orario, o in alcun giorni. 
o conviene andare su uno shelly o su sonoff? grazie

**Comment by pigioldHAOS** [comment]
Ciao, io posso dirti cosa ho integrato WOOX modello R7060 e funziona molto bene, accessorio basico. Recensito anche da Indomus se non sbaglio.

**Reply by reclus23** [reply]
ciao, grazie per il tuo intervento. Questo prodotto mi era sfuggito. Quindi è tranquillamente programmabile (giorni, orario e tempo di annaffiamento).
Ho fatto una ricerca e sui principali siti non c'è più.. che tu sappia è stato sostituito da una versione più recente ?

**Reply by pigioldHAOS** [reply]
Sono andato a vedere sul sito ufficiale e lì è disponibile.

**Reply by reclus23** [reply]
grazie!
